internal/controller/nnf_access_controller.go

```diff
@@ -1086,6 +1148,7 @@ func (r *NnfAccessReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		WithOptions(controller.Options{MaxConcurrentReconciles: maxReconciles}).
 		For(&nnfv1alpha2.NnfAccess{}).
+		Watches(&dwsv1alpha2.Computes{}, handler.EnqueueRequestsFromMapFunc(r.ComputesEnqueueRequests)).
```

Comment:
so this affectively triggers a reconcile with the NnfAccess controller?

Reply:
Yes, if the Computes resource for an NnfSystemStorage (or a Workflow) changes, the associated NnfAccess will reconcile again. For the Workflow, the Computes resource won't change, so this really only affects NnfSystemStorage.